CHANGELOG.md

```diff
@@ -0,0 +1,396 @@
+# Changelog
+
+## 2.35
+2019 Nov 01
+
+- Update durable-task (v1.33)
```

Comment:
nit: I wouldn't put this line in the changelog, since it doesn't really mean anything to users by itself, and a good portion of users are probably already running it anyway. IMO, the second line covers the change that users might care about in relation to the durable-task update.